src/Cedar/BridgeUnix.h

```diff
@@ -186,6 +186,7 @@ struct CAPTUREBLOCK{
 
 
 // Function prototype
+void FreeTap(VLAN *v);
```

Comment:
This function is implemented at src/Cedar/VLanUnix.c line 388. So the definition should also be under: src/Cedar/VLanUnix.h

Reply:
but the question is VLanUnix.c, the compiler find a undeclared function which defined in other file in fact. so it will warning "implicit declaration". Therefore I add a declaration before using void FreeTap(VLAN *).
is it necessary? you are export I following your advice.

see code:
https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/src/Cedar/BridgeUnix.c#L1570
FreeTap was define in other files. Books say  that if call the function defined in another file we should add function prototype before calling.